### ISSUE TITLE
Xcode Macbook works using free provisioning but VS windows all apps crash

### ISSUE THREAD

**Issue by DwainSnickles** [xamarin/Xamarin.Forms#14124]
### Description
All apps load, deploy and work from MacBook pro using Xcode and VS 2019 for Mac and deploying to my devices iPhone and iPad using Free Provisioning. When I switch to Windows VS 2109 16.9.3 and deploy I get no errors from VS 2019 and the app does deploy to my devices when I try to start the app on my iPhone or iPad it crashes instantly. I have been working on this for days now i have tried several different configurations but cant get it to work what am I missing here? I have been working with ColeXia he suggested I try to get help here. see https://docs.microsoft.com/en-us/answers/questions/345120/ios-apps-all-crash-when-using-visual-studio-2019.html?childToView=348894#comment-348894 Mac and PC are paired and I am using a blank default welcome to Xamarin forms blank application with no additional code added. Not sure if this helps but when trying to stop debugging in VS 2109 I always get this
![XSCapture](https://user-images.githubusercontent.com/26853477/114029135-d4c4e580-983e-11eb-96ee-a97c7d50263c.PNG)

### Steps to Reproduce

1. 
2. 
3. 

### Basic Information

Here is a list of what I changed in Xcode 12.4 Using the link above Click the Project>Projectname scroll down to Linking Click on Runpath Search paths do not manually type this in click until a text editor box is shown then click the + button and add each of these lines
$(inherited)
@executable_path/Frameworks
@loader_path/Frameworks

now click the Targets>Projectname scroll down to build Option change Always Embed Swift Standard Libraries make sure it says yes.
Scroll down to Linking Runpaths Search Paths and add the 3 lines as you did from above eg $(inherited).....
Scroll down to Signing make sure you have your team listed if not select your team.

removed the original app from iPad reloaded app from VS mac it now works on Ipad and iPhone.
Now onto VS2109 from windows will update you shortly.
Argh Still Crashes.
- Version with issue:
- Last known good version:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library / AndroidX Version: 
- NuGet Packages:
- Affected Devices:

### Environment

<details><summary>Show/Hide Visual Studio info</summary>

```
```

</details>

### Build Logs

Xccode Macbook all works including Visual Studio Mac I can successfully deploy to iPhone or iPad. When I try to use vs2019 windows 16.9.3 and deploy all apps crash on start up. It important to not that I have not added any code just using the default code from blank Xamarin application aka Welcome to Xamarin forms.  I will post he code here but some details on what I have tried are listed here https://docs.microsoft.com/en-us/answers/questions/345120/ios-apps-all-crash-when-using-visual-studio-2019.html?childToView=348894#comment-348894 my iPhone error code is as follows:
`{"app_name":"MobileTechnician.iOS","timestamp":"2021-04-04 17:58:18.00 -0500","app_version":"1.0","slice_uuid":"9dc5b807-6c36-33d8-b086-033235042e47","adam_id":0,"build_version":"1.0","platform":2,"bundleID":"com.Dwain.MobileTechnician","share_with_app_devs":0,"is_first_party":0,"bug_type":"109","os_version":"iPhone OS 14.4.2 (18D70)","incident_id":"BA621376-5DB6-4192-9C3A-0C6D5AB495CE","name":"MobileTechnician.iOS"}
Incident Identifier: BA621376-5DB6-4192-9C3A-0C6D5AB495CE
CrashReporter Key: 05dc789dd2daf05d495a5c8596e544713c001e3b
Hardware Model: iPhone12,3
Process: MobileTechnician.iOS [14584]
Path: /private/var/containers/Bundle/Application/D992F17B-4950-428D-B228-4D92A84F9976/MobileTechnician.iOS.app/MobileTechnician.iOS
Identifier: com.Dwain.MobileTechnician
Version: 1.0 (1.0)
Code Type: ARM-64 (Native)
Role: Foreground
Parent Process: launchd [1]
Coalition: com.Dwain.MobileTechnician [504]

Date/Time: 2021-04-04 17:58:18.9256 -0500
Launch Time: 2021-04-04 17:58:18.9184 -0500
OS Version: iPhone OS 14.4.2 (18D70)
Release Type: User
Baseband Version: 2.03.04
Report Version: 104

Exception Type: EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note: EXC_CORPSE_NOTIFY
Termination Description: DYLD, Library not loaded: @rpath/libMobileTechnician.iOS.exe.dylib | Referenced from: /private/var/containers/Bundle/Application/D992F17B-4950-428D-B228-4D92A84F9976/MobileTechnician.iOS.app/MobileTechnician.iOS | Reason: no suitable image found. Did find: | /private/var/containers/Bundle/Application/D992F17B-4950-428D-B228-4D92A84F9976/MobileTechnician.iOS.app/libMobileTechnician.iOS.exe.dylib: code signature in (/private/var/containers/Bundle/Application/D992F17B-4950-428D-B228-4D92A84F9976/MobileTechnician.iOS.app/libMobileTechnician.iOS.exe.dylib) not valid for use in process using Library Validation: mapped file has no cdhash, completely unsigned? Code has to be at least ad-hoc signed. | /private/var/containers/Bundle/Application/D992F17B-4950-428D-B228-4D92A84F9976/MobileTechnician.iOS.app/libMobileTechnician.iOS.exe.dylib: stat() failed with errno=1
Triggered by Thread: 0

Thread 0 Crashed:
0 dyld 0x000000010255f440 0x102508000 + 357440
1 dyld 0x00000001025676e0 0x102508000 + 390880
2 dyld 0x0000000102567714 0x102508000 + 390932
3 dyld 0x000000010250d05c 0x102508000 + 20572
4 dyld 0x00000001025100b8 0x102508000 + 32952
5 dyld 0x0000000102509258 0x102508000 + 4696
6 dyld 0x0000000102509038 0x102508000 + 4152

Thread 0 crashed with ARM Thread State (64-bit):
x0: 0x0000000000000006 x1: 0x0000000000000005 x2: 0x000000016db76af0 x3: 0x00000000000000bb
x4: 0x000000016db766f0 x5: 0x0000000000000000 x6: 0x0000000000000000 x7: 0x00000000000000f0
x8: 0x0000000000000020 x9: 0x0000000000000009 x10: 0x0000000000000001 x11: 0x00000001b79be000
x12: 0x0000000000004018 x13: 0x00000001f54b9f13 x14: 0x0000000000000005 x15: 0x000000016db6d8a0
x16: 0x0000000000000209 x17: 0x00000001025597b4 x18: 0x0000000000000000 x19: 0x0000000000000000
x20: 0x000000016db766f0 x21: 0x00000000000000bb x22: 0x000000016db76af0 x23: 0x0000000000000005
x24: 0x0000000000000006 x25: 0x000000016db77808 x26: 0x0000000102580000 x27: 0x0000000102580000
x28: 0x00000001024c8000 fp: 0x000000016db766c0 lr: 0x00000001025676e0
sp: 0x000000016db76680 pc: 0x000000010255f440 cpsr: 0x00000000
esr: 0x56000080 Address size fault

Binary Images:
0x102288000 - 0x1023c7fff MobileTechnician.iOS arm64 <9dc5b8076c3633d8b086033235042e47> /var/containers/Bundle/Application/D992F17B-4950-428D-B228-4D92A84F9976/MobileTechnician.iOS.app/MobileTechnician.iOS
0x102508000 - 0x102577fff dyld arm64e <1f4d7499ee603c5d9d542cd29e4e537c> /usr/lib/dyld
0x185708000 - 0x185ac1fff CoreFoundation arm64e <727f2644eb4e3d57bc2ee6803ba92366> /System/Library/Frameworks/CoreFoundation.framework/CoreFoundation
0x185cb7000 - 0x185d33fff SystemConfiguration arm64e <97ad4485538f3f389fcdb3fb4df441c3> /System/Library/Frameworks/SystemConfiguration.framework/SystemConfiguration
0x185d34000 - 0x185e2bfff CoreTelephony arm64e <810c9ed7d97438e19ca18fac779bab84> /System/Library/Frameworks/CoreTelephony.framework/CoreTelephony
0x185e2c000 - 0x1862c3fff CFNetwork arm64e <0ebb3119a8ce35da96245b5528802fc8> /System/Library/Frameworks/CFNetwork.framework/CFNetwork
0x1869d2000 - 0x186a48fff Accounts arm64e <8cc93a19e5983eeb8afb506477688616> /System/Library/Frameworks/Accounts.framework/Accounts
0x186a49000 - 0x186d04fff Foundation arm64e <7698bf3e0cf631c085e9562714f01276> /System/Library/Frameworks/Foundation.framework/Foundation
0x186d05000 - 0x187067fff ImageIO arm64e <b557d05ab2a53d2eb43e0db267169596> /System/Library/Frameworks/ImageIO.framework/ImageIO
0x187081000 - 0x187619fff CoreGraphics arm64e <f5b44b3793af3d1fb053789a8626498a> /System/Library/Frameworks/CoreGraphics.framework/CoreGraphics
0x188aeb000 - 0x188d71fff QuartzCore arm64e <8510f13908243686a9aa3e198539a021> /System/Library/Frameworks/QuartzCore.framework/QuartzCore
0x1899c6000 - 0x189bedfff ContactsUI arm64e <27576b968fb33ee08ae4009687396b75> /System/Library/Frameworks/ContactsUI.framework/ContactsUI
0x189bee000 - 0x189da5fff CoreText arm64e <58b7adf717bf3fb093fd084cb70cba54> /System/Library/Frameworks/CoreText.framework/CoreText
0x18a087000 - 0x18a1e9fff Contacts arm64e <387594fd3d2c321183e16487cc92d086> /System/Library/Frameworks/Contacts.framework/Contacts
0x18b740000 - 0x18b7d5fff CoreLocation arm64e <27c07cfbfe903a308300038163f75a62> /System/Library/Frameworks/CoreLocation.framework/CoreLocation
0x18b7d6000 - 0x18b988fff CloudKit arm64e <91285302d0043991af90fe2b0f3727d0> /System/Library/Frameworks/CloudKit.framework/CloudKit
0x18b989000 - 0x18bd07fff CoreData arm64e <e4983c77f15e3012bf00a3fc6f2eb012> /System/Library/Frameworks/CoreData.framework/CoreData
0x18cd6c000 - 0x18ceb9fff Security arm64e <248431aabfe8399dad0ea942783d0e4b> /System/Library/Frameworks/Security.framework/Security
0x18ceba000 - 0x18d2ddfff MediaPlayer arm64e <e7f1d156c6923b30906faf6183ed22e8> /System/Library/Frameworks/MediaPlayer.framework/MediaPlayer
0x18d498000 - 0x18da44fff Intents arm64e <397b9f43ab043a72bc5ca59dfddd48fa> /System/Library/Frameworks/Intents.framework/Intents
0x18da94000 - 0x18ddadfff CoreImage arm64e <bbd5ff0b4792336fb8482f5c0c6c692c> /System/Library/Frameworks/CoreImage.framework/CoreImage
0x18de6c000 - 0x18dea7fff CoreVideo arm64e <a41932f1812e3928b8188feef048018d> /System/Library/Frameworks/CoreVideo.framework/CoreVideo
0x18dea8000 - 0x18e694fff MediaToolbox arm64e <334f787e71c73676b9136c5f2a3436dc> /System/Library/Frameworks/MediaToolbox.framework/MediaToolbox
0x18e695000 - 0x18e795fff CoreMedia arm64e <063231877fe13d2d9c430fdad845e776> /System/Library/Frameworks/CoreMedia.framework/CoreMedia
0x18e796000 - 0x18ea01fff AudioToolbox arm64e <62b836837f2237f89cd6aed24461df2c> /System/Library/Frameworks/AudioToolbox.framework/AudioToolbox
0x18ea02000 - 0x18ea52fff CoreHaptics arm64e <bbd7402fc1b33e1a9258307051360494> /System/Library/Frameworks/CoreHaptics.framework/CoreHaptics
0x18ecb0000 - 0x18fdeafff JavaScriptCore arm64e <6142416206cc3018bd316cac8586e04f> /System/Library/Frameworks/JavaScriptCore.framework/JavaScriptCore
0x19044e000 - 0x19069cfff HealthKit arm64e <d7a313170d383d33bc7736dd1660fa02> /System/Library/Frameworks/HealthKit.framework/HealthKit
0x1908f9000 - 0x190b6cfff CoreMotion arm64e <dd8e0d47a8cd3194a28b3bb6dfc7253c> /System/Library/Frameworks/CoreMotion.framework/CoreMotion
0x190b6d000 - 0x190cb2fff EventKit arm64e <374caf4729133e16933429279ffba9ca> /System/Library/Frameworks/EventKit.framework/EventKit
0x19166d000 - 0x1916ddfff CoreSpotlight arm64e <22dc7b9c417e35459569bcc9bb02cadb> /System/Library/Frameworks/CoreSpotlight.framework/CoreSpotlight
0x191dbe000 - 0x1927b7fff WebKit arm64e <b99e650dc7db369fa4b6183b7a8b86e4> /System/Library/Frameworks/WebKit.framework/WebKit
0x195558000 - 0x19558bfff UserNotifications arm64e <afe58449cf0234a4bc234ded3712a3d5> /System/Library/Frameworks/UserNotifications.framework/UserNotifications
0x19563d000 - 0x195918fff MapKit arm64e <3108aac5d11938e09aaff67868010ade> /System/Library/Frameworks/MapKit.framework/MapKit
0x196bf7000 - 0x196e47fff Photos arm64e <fb8fe08f174636eeb91169c0f87ae460> /System/Library/Frameworks/Photos.framework/Photos
0x1976da000 - 0x19792bfff NetworkExtension arm64e <7f3e9729ca2e39ada585c363f6a7a9dc> /System/Library/Frameworks/NetworkExtension.framework/NetworkExtension
0x197963000 - 0x197d2ffff CoreML arm64e <f9033ece3eeb35079edb4ea0a1384705> /System/Library/Frameworks/CoreML.framework/CoreML
0x1984b6000 - 0x1985b3fff VideoToolbox arm64e <a291cd84788c36a095d8a7e53275e4cd> /System/Library/Frameworks/VideoToolbox.framework/VideoToolbox
0x198711000 - 0x198cdffff PhotosUI arm64e <90585de26df539e3aba01ad2ef94a5c5> /System/Library/Frameworks/PhotosUI.framework/PhotosUI
0x199d47000 - 0x199e27fff LinkPresentation arm64e <d43f5fbb6c8d39cb8406e5cf98d4b578> /System/Library/Frameworks/LinkPresentation.framework/LinkPresentation
0x199fdb000 - 0x199fecfff UniformTypeIdentifiers arm64e <15720249377a3189b5acde89ce28022a> /System/Library/Frameworks/UniformTypeIdentifiers.framework/UniformTypeIdentifiers
0x19a2f0000 - 0x19a570fff HomeKit arm64e <aff5c4488b4d3352b6c94852175254e6> /System/Library/Frameworks/HomeKit.framework/HomeKit
0x19ab26000 - 0x19ac84fff Network arm64e <9d7e4a86670f3742b6c188f150e74705> /System/Library/Frameworks/Network.framework/Network
0x19b265000 - 0x19b37ffff FileProvider arm64e <d9607b48e828313d88733480c22fbc28> /System/Library/Frameworks/FileProvider.framework/FileProvider
0x19bcad000 - 0x19bdbafff AVKit arm64e <5ef8c42722333fbbbfe902dda8002e0f> /System/Library/Frameworks/AVKit.framework/AVKit
0x19bdbb000 - 0x19bdc3fff BackgroundTasks arm64e <1e7df2f7f5d73763bc2e2ae1849dfa37> /System/Library/Frameworks/BackgroundTasks.framework/BackgroundTasks
0x19bfba000 - 0x19c09afff Metal arm64e <b19644ec655d330fb5022316136a7b66> /System/Library/Frameworks/Metal.framework/Metal
0x19ce19000 - 0x19cf57fff MessageUI arm64e <f25c08dbdf4e31cf8bda2c6a0a99e00c> /System/Library/Frameworks/MessageUI.framework/MessageUI
0x19ee94000 - 0x19f140fff Vision arm64e <c5ba507230a53c1f9b161e9c08e24b02> /System/Library/Frameworks/Vision.framework/Vision
0x19f473000 - 0x19f4d2fff CoreBluetooth arm64e <07157e457e543ae4ad87685503c2db42> /System/Library/Frameworks/CoreBluetooth.framework/CoreBluetooth
0x1a0434000 - 0x1a0619fff EventKitUI arm64e <60a30e6e93623df5a241c3e6da6b5508> /System/Library/Frameworks/EventKitUI.framework/EventKitUI
0x1a09f3000 - 0x1a0be1fff SafariServices arm64e <65633f0d99f23bb7a39bc68fad864fb9> /System/Library/Frameworks/SafariServices.framework/SafariServices
0x1a17cf000 - 0x1a188dfff QuickLook arm64e <9b078bce485a3f168dd324e34ce1b03d> /System/Library/Frameworks/QuickLook.framework/QuickLook
0x1a3b56000 - 0x1a3bcdfff ClassKit arm64e <df0f8ebb071c3654aeeb6a7fb095eb44> /System/Library/Frameworks/ClassKit.framework/ClassKit
0x1a3bce000 - 0x1a3c15fff StoreKit arm64e <c15dd601bcba3a5f815dcc6a70b50b80> /System/Library/Frameworks/StoreKit.framework/StoreKit
0x1a50fe000 - 0x1a5168fff CallKit arm64e <40172deb9a033fdbb2f9c19402e18198> /System/Library/Frameworks/CallKit.framework/CallKit
0x1a538a000 - 0x1a5648fff PencilKit arm64e <2ac11145139332a985805ff6843ec804> /System/Library/Frameworks/PencilKit.framework/PencilKit
0x1a79b4000 - 0x1a79c6fff IOSurface arm64e <2bb4143452ee3bd7822950b3840218b6> /System/Library/Frameworks/IOSurface.framework/IOSurface
0x1aa03e000 - 0x1aa0a6fff iAd arm64e <ba0cf267b59b327aa9ca1e3bd4a1ca49> /System/Library/Frameworks/iAd.framework/iAd
0x1af95e000 - 0x1af965fff PushKit arm64e <ea9b2ff9825f30a29dc38fe3e70fd6de> /System/Library/Frameworks/PushKit.framework/PushKit
0x1b0c12000 - 0x1b0c53fff NaturalLanguage arm64e <fd73fd92ced23b1ab02c6ffdee31eece> /System/Library/Frameworks/NaturalLanguage.framework/NaturalLanguage
0x1b2962000 - 0x1b298efff SensorKit arm64e <3b374d9b78ae3ac08b53bb0bd3d59b94> /System/Library/Frameworks/SensorKit.framework/SensorKit
0x1b2e85000 - 0x1b2e9efff LocalAuthentication arm64e <a99dfb53f24d3af59e419be0d7d32ec8> /System/Library/Frameworks/LocalAuthentication.framework/LocalAuthentication
0x1b431a000 - 0x1b4326fff MediaAccessibility arm64e <babd3b53c8f63359aa5a98e3bfec3f34> /System/Library/Frameworks/MediaAccessibility.framework/MediaAccessibility
0x1b493d000 - 0x1b499ffff Social arm64e <1b58248fbc8838e59709781d5a079700> /System/Library/Frameworks/Social.framework/Social
0x1b6481000 - 0x1b6498fff MediaSetup arm64e <ca887fb49bee36d9b137eb8b492f7aa4> /System/Library/Frameworks/MediaSetup.framework/MediaSetup
0x1b702e000 - 0x1b7069fff ARKit arm64e <964ad61cb22a3237941536dfac9fd07d> /System/Library/Frameworks/ARKit.framework/ARKit
0x1b706a000 - 0x1b706afff AVFoundation arm64e <1dcd460171643d2ca077cc511181b1ec> /System/Library/Frameworks/AVFoundation.framework/AVFoundation
0x1b706b000 - 0x1b706bfff Accelerate arm64e <b2f811f636d9398b9e2a080408efed72> /System/Library/Frameworks/Accelerate.framework/Accelerate
0x1b785b000 - 0x1b785efff Accessibility arm64e <1994de9b5c6f3f40b2485f066f2fb512> /System/Library/Frameworks/Accessibility.framework/Accessibility
0x1b785f000 - 0x1b7860fff AdSupport arm64e <aa94023647e73d2f9a83acfa23a5a8f3> /System/Library/Frameworks/AdSupport.framework/AdSupport
0x1b7861000 - 0x1b788afff AddressBook arm64e <c14193c94af236af8bb62ea248a770eb> /System/Library/Frameworks/AddressBook.framework/AddressBook
0x1b788b000 - 0x1b7892fff AddressBookUI arm64e <2ce9d990889e32909f077c4f9dc3fb3a> /System/Library/Frameworks/AddressBookUI.framework/AddressBookUI
0x1b7893000 - 0x1b7894fff AppClip arm64e <db56dd5989f13da79fff0b89f6f1f8b7> /System/Library/Frameworks/AppClip.framework/AppClip
0x1b7895000 - 0x1b7896fff AppTrackingTransparency arm64e <291109c01d4130c0b0080b4acf5238f6> /System/Library/Frameworks/AppTrackingTransparency.framework/AppTrackingTransparency
0x1b7897000 - 0x1b78a8fff AssetsLibrary arm64e <5ed3398a0a533c789bace00f231f2d88> /System/Library/Frameworks/AssetsLibrary.framework/AssetsLibrary
0x1b79be000 - 0x1b79dbfff AuthenticationServices arm64e <c926d27ed94a3eefa7b67990ceca4a43> /System/Library/Frameworks/AuthenticationServices.framework/AuthenticationServices
0x1b79dc000 - 0x1b79defff AutomaticAssessmentConfiguration arm64e <6987e4fddcdf3435be91ba6023697249> /System/Library/Frameworks/AutomaticAssessmentConfiguration.framework/AutomaticAssessmentConfiguration
0x1b79ec000 - 0x1b7a07fff BusinessChat arm64e <c011500a1573309d8f183185c71590b6> /System/Library/Frameworks/BusinessChat.framework/BusinessChat
0x1b7a08000 - 0x1b7a46fff CarPlay arm64e <dba397a596463ced83090bd12af2520d> /System/Library/Frameworks/CarPlay.framework/CarPlay
0x1b7ac1000 - 0x1b7ad3fff CoreAudioKit arm64e <e669b1e6756c386e9198d23b7357794c> /System/Library/Frameworks/CoreAudioKit.framework/CoreAudioKit
0x1b7ad4000 - 0x1b7b3dfff CoreMIDI arm64e <9411ca7963a431d7835a511ff2e3fcd3> /System/Library/Frameworks/CoreMIDI.framework/CoreMIDI
0x1b7b3e000 - 0x1b7b6cfff CoreNFC arm64e <b85dce09617a3590a4bdeb0107c5d292> /System/Library/Frameworks/CoreNFC.framework/CoreNFC
0x1b806d000 - 0x1b8071fff DeviceCheck arm64e <058bceb5705634cf845dc41b29e6faaa> /System/Library/Frameworks/DeviceCheck.framework/DeviceCheck
0x1b8072000 - 0x1b8089fff ExternalAccessory arm64e <af3bd935704c3326a38f725bd2a01142> /System/Library/Frameworks/ExternalAccessory.framework/ExternalAccessory
0x1b808a000 - 0x1b809dfff FileProviderUI arm64e <273430f12b4635f385f565b6937f4aa8> /System/Library/Frameworks/FileProviderUI.framework/FileProviderUI
0x1b809e000 - 0x1b80c7fff GLKit arm64e <fe523fc12e733ba6ac0dce7bef9ab320> /System/Library/Frameworks/GLKit.framework/GLKit
0x1b80f6000 - 0x1b819efff GameController arm64e <f60f3177298136f7bfd784b5fa678ed6> /System/Library/Frameworks/GameController.framework/GameController
0x1b819f000 - 0x1b819ffff GameKit arm64e <1450bf78c83239638ee60d92ef16dec8> /System/Library/Frameworks/GameKit.framework/GameKit
0x1b81a0000 - 0x1b8209fff GameplayKit arm64e <ce01e386abe837cc9e9e503980aa8ef1> /System/Library/Frameworks/GameplayKit.framework/GameplayKit
0x1b820a000 - 0x1b8213fff HealthKitUI arm64e <7c40a31e11023945afe66ad5b8112cf7> /System/Library/Frameworks/HealthKitUI.framework/HealthKitUI
0x1b8214000 - 0x1b8220fff IdentityLookup arm64e <cba22d6066113d8e99539205b9e416de> /System/Library/Frameworks/IdentityLookup.framework/IdentityLookup
0x1b8221000 - 0x1b8228fff IdentityLookupUI arm64e <003af54a98f93c3388c11071ca49289b> /System/Library/Frameworks/IdentityLookupUI.framework/IdentityLookupUI
0x1b8229000 - 0x1b8241fff IntentsUI arm64e <0ebc4655e56432ce8c9615e4a0c8f068> /System/Library/Frameworks/IntentsUI.framework/IntentsUI
0x1b8262000 - 0x1b83b5fff MLCompute arm64e <238c5cacb32d3a82b319be8226d72517> /System/Library/Frameworks/MLCompute.framework/MLCompute
0x1b83bd000 - 0x1b83ddfff Messages arm64e <8fa3ed535fe834308cad2e6c1c736d64> /System/Library/Frameworks/Messages.framework/Messages
0x1b83de000 - 0x1b83f7fff MetalKit arm64e <e2a1b550d07a35c68ba7321e53f7531e> /System/Library/Frameworks/MetalKit.framework/MetalKit
0x1b852e000 - 0x1b852efff MetalPerformanceShaders arm64e <c3e2c89e6c6b35a79c4acc3c0c8f817a> /System/Library/Frameworks/MetalPerformanceShaders.framework/MetalPerformanceShaders
0x1b88b8000 - 0x1b88cafff MetricKit arm64e <546fef994f4b3bdaa48e3600a36e4052> /System/Library/Frameworks/MetricKit.framework/MetricKit
0x1b88cb000 - 0x1b88cbfff MobileCoreServices arm64e <013cab54d0ec301a82151c17a2a44899> /System/Library/Frameworks/MobileCoreServices.framework/MobileCoreServices
0x1b88cc000 - 0x1b934bfff ModelIO arm64e <83f129a77c5f327c821057dce70da9c6> /System/Library/Frameworks/ModelIO.framework/ModelIO
0x1b934c000 - 0x1b9391fff MultipeerConnectivity arm64e <6e898f3373143680800f0206bc6c4571> /System/Library/Frameworks/MultipeerConnectivity.framework/MultipeerConnectivity
0x1b9392000 - 0x1b93a4fff NearbyInteraction arm64e <acf64110f29b379ba18d28f300957c9e> /System/Library/Frameworks/NearbyInteraction.framework/NearbyInteraction
0x1b93a5000 - 0x1b93aefff NewsstandKit arm64e <c5b1f3528a1a35f9ba5f185d2b76c038> /System/Library/Frameworks/NewsstandKit.framework/NewsstandKit
0x1b93af000 - 0x1b93bcfff NotificationCenter arm64e <250f04db70b43a1db5df6d7934113e17> /System/Library/Frameworks/NotificationCenter.framework/NotificationCenter
0x1b94c9000 - 0x1b94d1fff OpenGLES arm64e <fa5413d895ef3dada5eed752ca32bb1f> /System/Library/Frameworks/OpenGLES.framework/OpenGLES
0x1b96a5000 - 0x1b976cfff PDFKit arm64e <333f6e509f53397e9ad43cd0c255e123> /System/Library/Frameworks/PDFKit.framework/PDFKit
0x1b976d000 - 0x1b977bfff PassKit arm64e <77e019e8d47f3d58b7f33cfc1ef42085> /System/Library/Frameworks/PassKit.framework/PassKit
0x1b977c000 - 0x1b97bafff QuickLookThumbnailing arm64e <b5eb6198584d3520aadb6283de3848c7> /System/Library/Frameworks/QuickLookThumbnailing.framework/QuickLookThumbnailing
0x1b9a27000 - 0x1b9a56fff ReplayKit arm64e <ec793c07951838e8950bf471db348b81> /System/Library/Frameworks/ReplayKit.framework/ReplayKit
0x1b9a57000 - 0x1b9e85fff SceneKit arm64e <ec930bc71d623c34ae79b49a41f6198e> /System/Library/Frameworks/SceneKit.framework/SceneKit
0x1b9e86000 - 0x1b9e8cfff ScreenTime arm64e <5122183593ab3d8ba517b57d857db506> /System/Library/Frameworks/ScreenTime.framework/ScreenTime
0x1b9e8d000 - 0x1b9fdafff SoundAnalysis arm64e <df974c1d35fe3485af2bfe0d53f4ec62> /System/Library/Frameworks/SoundAnalysis.framework/SoundAnalysis
0x1b9fdb000 - 0x1b9feafff Speech arm64e <fbb15fe119ed3cf88c4d750dfc822868> /System/Library/Frameworks/Speech.framework/Speech
0x1b9feb000 - 0x1ba10bfff SpriteKit arm64e <b694cfae6f2d3d1981bcb1c33991802d> /System/Library/Frameworks/SpriteKit.framework/SpriteKit
0x1ba10c000 - 0x1ba10dfff Twitter arm64e <4dbec502d2a63c4291ae9603c4414e13> /System/Library/Frameworks/Twitter.framework/Twitter
0x1ba10e000 - 0x1ba10efff UIKit arm64e <d6833c925bf638189ede3bb4092a0919> /System/Library/Frameworks/UIKit.framework/UIKit
0x1ba10f000 - 0x1ba117fff UserNotificationsUI arm64e <9d8ca0d2586f3285a401069d2cebc4a2> /System/Library/Frameworks/UserNotificationsUI.framework/UserNotificationsUI
0x1ba118000 - 0x1ba1e1fff VideoSubscriberAccount arm64e <130a847f2a033dfb9eb7379e87aff600> /System/Library/Frameworks/VideoSubscriberAccount.framework/VideoSubscriberAccount
0x1badd4000 - 0x1badd4fff VisionKit arm64e <060851a2993e308f94396a3c54f6655e> /System/Library/Frameworks/VisionKit.framework/VisionKit
0x1badd5000 - 0x1bae11fff WatchConnectivity arm64e <a518167a64053c0593597ec7d578cbdb> /System/Library/Frameworks/WatchConnectivity.framework/WatchConnectivity
0x1d29b9000 - 0x1d29bafff AdServices arm64e <d771666d4e203e119fd9f40fdcd2218a> /System/Library/Frameworks/AdServices.framework/AdServices

EOF`

**Comment by DwainSnickles** [comment]
I keep trying here is somethin I took the sample ToDo app from Xamarin samples on github Created a Xcode for signing I deploy it to my iphone and get the following error in VS2109 
`Severity	Code	Description	Project	File	Line	Suppression State
Error		error MT1007: Failed to launch the application '/Users/dwainsnickles/Library/Caches/Xamarin/mtbs/builds/Todo.iOS/4aa98bdb0386ce3d626197ec74b9b3e88ccbb573d27d8b5142908c72d12fa020/bin/iPhone/Debug/device-builds/iphone12.3-14.4.2/QuickTodoiOS.app' on the device 'dwain???s iPhone': Failed to launch the application 'com.Dwain.ToDo' on the device 'dwain???s iPhone': Invalid Service Error (error: 0xe8000022)		 	0	
`

**Comment by DwainSnickles** [comment]
Yeah after much frustration I figured it out not sure why or how but originally I selected Dont link in IOS properties because VS told me there was a problem with Hot Reload the app the worked was set to link framework sdk only I switched back and rebuilt my old project and it worked went back to don't link after successful run as i do want to use hot reload. Now it works not sure why and advice would be helpful. Persistence is my strongest suit.
